client/my-sites/pages/page/index.js

```diff
@@ -421,42 +430,151 @@ class Page extends Component {
 						siteUrl={ this.props.multisite && this.getSiteDomain() }
 					/>
 				</div>
-				{ ellipsisGridicon }
-				{ popoverMenu }
-				<ReactCSSTransitionGroup
-					transitionName="updated-trans"
-					transitionEnterTimeout={ 300 }
-					transitionLeaveTimeout={ 300 }
-				>
-					{ this.props.buildUpdateTemplate() }
-				</ReactCSSTransitionGroup>
+				{ ellipsisMenu }
+				{ this.renderShadowNotice() }
```

Comment:
total nit: don't love functions named `renderX()`.  I usually tend to make a little Component outside of the class definition.

```jsx

function shadowNotice( props ) {
 return ....;
}

class Page extends Component {
  render() {
     { this.props.shadow.shadowStatus && <ShadowNotice { relevantPropsHere } /> }
  }
}
```

Reply:
OK, I extracted a `ShadowNotice` component.